test/dspal_tester/adsp_proc/posix_time_tests.c

```diff
@@ -257,8 +265,108 @@ int dspal_tester_test_usleep(void)
 		LOG_ERR("usleep run length: %d", length);
 		FAIL("usleep returned too fast");
 	}
+	return TEST_PASS;
+}
+
+static void test_dspal_get_time_1us()
+{
+	uint64_t start_time = dspal_get_time_1us();
+	int counter = 5000;
+	for (int i = 0; i < counter; i ++) {		
+		uint64_t curr_time = dspal_get_time_1us();
+	}
+	int elapsed_time = dspal_get_time_1us() - start_time;
+	LOG_ERR("total %d dspal_get_time_1us calls takes %d, each %f", counter, elapsed_time, elapsed_time*1.0/counter);
+	
+}
+int test_usleep_worker(int flag)
```

Comment:
Need to document expected latency, that threads are sched FIFO, and any thread at the same priority or higher can create latency.